### DISCUSSION TITLE
Arduino ESP32 PNG display example from within a class

### DISCUSSION THREAD

**Discussion by MikeysLab** [Bodmer/TFT_eSPI#3052] [General]
Is there an example of displaying a PNG from LittleFS for the ESP32 being implimented inside of a class. Having issues with the pngDraw callback.
![image](https://github.com/Bodmer/TFT_eSPI/assets/40308634/c168decf-eef2-4a6e-bebe-97b1bb56677c)
![image](https://github.com/Bodmer/TFT_eSPI/assets/40308634/2c0d0513-72e7-4423-a345-bc58728b3587)

**Comment by Bodmer** [comment]
There is not example for this. You will need to make the callback function static to use it as a callback. This is used callback in a class approach is used in [this library](https://github.com/Bodmer/TJpg_Decoder) and [this one](https://github.com/Bodmer/JPEGDecoder) too,
